examples/turbulent_advection-diffusion.jl

```diff
@@ -131,7 +134,7 @@ saveproblem(output)
 # We specify that we would like to save the concentration every `save_frequency` timesteps;
```

Comment:
Add `nothing # hide` on line 85, 97 and 101?

Reply:
When the output is small then I like to show it. E.g., the output from constructing the `ADprob`..